docs/uefi-db.md

```diff
@@ -0,0 +1,203 @@
+---
+title: UEFI Secure Boot Certificates
+---
+
+## Executive Summary
+
+Very soon a certificate we use for signing boot media will expire. We recently found out that Microsoft will not sign updated boot media with the old key, and that **at least one tier-1 OEM** is not going to be shipping the expired key on new hardware. This means that existing install media may not boot on some new laptop, desktop and server devices, and that future updates to boot packages may not boot on old hardware.
+
+Microsoft is shipping fixes for select OEMs using Windows Updates automatically. The workaround for Linux is to manually disable secure boot which would be unpopular with anyone that cares about security. We need to build a solution that can distribute the updated certificates in Linux.
+
+## Important Terms
+
+* `PK`: “*Platform Key*” – one X509 certificate created by the OEM, e.g. Lenovo
+* `KEK`: “*Key Exchange Key*” – multiple certificates (created by Microsoft and the OEM), used to update db, signed by the vendor PK
+* `db`: “*Signature Database*” – multiple certificates (created by Microsoft and the OEM), hashes, and signatures, used to allow binaries, signed by KEK
+* `dbx`: “*Forbidden Signatures Database*” – multiple certificates, hashes, and signatures, used to block binaries, signed by KEK
+* 3rd party certificate – the certificate Microsoft uses to sign non-MS bootloaders that we use in Linux, e.g. Shim
+  * Windows Production CA 2011 – the old “PCA” certificate
+  * Microsoft UEFI CA 2023 (third-party) – the new certificate
```

Comment:
What do you think about using a markdown table instead?

Reply:
Bleh, I hate the markdown format. Lets stick to bullets for now and save tables for tabular data.